### PR TITLE
Fix: correct minor bugs in the import/export workflow

### PR DESCRIPTION
**Pull request by Rom1-B**
## Checklist before requesting a review

*Please delete options that are not relevant.*

- [x] I have performed a self-review of my code.
- [ ] I have added tests (when available) that prove my fix is effective or that my feature works.
- [x] I have updated the CHANGELOG with a short functional description of the fix or new feature.
- [ ] This change requires a documentation update.

## Description

- fixes #N/A
- Corrects several minor inconsistencies in the import and export workflow.
- Improves the reliability of checks and data handling used when running and configuring injection models.

### CODE REVIEW

File: inc/clientinjection.class.php [hunk 1]
```diff
@@ -320,6 +320,14 @@ public static function showResultsForm(PluginDatainjectionModel $model)
         TemplateRenderer::getInstance()->display('@datainjection/clientinjection_result.html.twig', $data);
     }
 
+    private static function escapeCsvFormula($value)
```
Comment (stonebuzz) [inline inc/clientinjection.class.php]: New security function with no unit test. It is a pure function with several edge cases worth pinning (empty string, non-string passthrough, each of the four trigger characters, a value starting with a safe character). Without a test, a future refactor or PHP upgrade could silently break the protection and let formula-injection payloads through.

Comment (stonebuzz) [reply inc/clientinjection.class.php]: Missing PHP 8.2 type declarations. The function accepts any type (non-strings are passed through unchanged) and returns the same type. Project rules require strict typing everywhere.

```suggestion
    private static function escapeCsvFormula(mixed $value): mixed
```

File: inc/clientinjection.class.php [hunk 2]
```diff
@@ -320,6 +320,14 @@ public static function showResultsForm(PluginDatainjectionModel $model)
...
+    {
+        if (is_string($value) && isset($value[0]) && str_contains('=+-@', $value[0])) {
```
Comment (stonebuzz) [inline inc/clientinjection.class.php]: Arguments are in an unconventional order: `str_contains` reads as "does haystack contain needle", so `str_contains('=+-@', $value[0])` means "does '=+-@' contain the first char?"
which is correct but non-obvious. `in_array` makes the intent explicit and self-documents the set of trigger characters.


```suggestion
        if (is_string($value) && isset($value[0]) && in_array($value[0], ['=', '+', '-', '@'], true)) {
```

**Review by stonebuzz — CHANGES_REQUESTED** [review]

**Review by stonebuzz — APPROVED** [review]
